aws-cpp-sdk-core/source/http/curl/CurlHttpClient.cpp

```diff
@@ -427,6 +431,35 @@ void CurlHttpClient::MakeRequestInternal(HttpRequest& request,
                 curl_easy_setopt(connectionHandle, CURLOPT_PROXYUSERNAME, m_proxyUserName.c_str());
                 curl_easy_setopt(connectionHandle, CURLOPT_PROXYPASSWORD, m_proxyPassword.c_str());
             }
+#if LIBCURL_VERSION_MAJOR >= 7
+#if LIBCURL_VERSION_MINOR >= 52
```

Comment:
Your PR probably precedes this change, but can you follow the pattern used here:
https://github.com/aws/aws-sdk-cpp/blob/master/aws-cpp-sdk-core/CMakeLists.txt#L94 and define `CURL_HAS_TLS_PROXY` and check that instead here.